### PR TITLE
renamed Url to URL to resolve lint warning and consistency

### PR DESCRIPTION
**Pull request by girishramnani**
**What kind of PR is this?**

/kind cleanup

**What does does this PR do / why we need it**:
see $SUBJECT

**Which issue(s) this PR fixes**:

No issue

**How to test changes / Special notes to the reviewer**:
just internal renaming of variable, tests should pass

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -74,11 +74,11 @@ type ComponentSettings struct {
 
 	Envs EnvVarList `yaml:"Envs,omitempty"`
 
-	Url *[]ConfigUrl `yaml:"Url,omitempty"`
+	URL *[]ConfigURL `yaml:"Url,omitempty"`
```
Comment (girishramnani) [inline pkg/config/config.go]: @kadel do we wanna change the `yaml:Url` to `yaml:URL`. This will break the backward compatibility of the config file.

Comment (kadel) [reply pkg/config/config.go]: lets keep it as it is

**Comment by dharmit** [comment]
Since #2362 got merged after the latest CI failure on this PR, simply re-triggering the 4.3 integration tests.

/test v4.3-integration

**Comment by girishramnani** [comment]
Terraform errors
/retest

**Pull request merged**